TheTechTravis/nss-aquarium
# Created three new functions in FishDataProvider.js to filter fish by size. Tested via console.log() and verified each of these functions works as intended. fish.length types were changed from String to Number and values were modified for this lesson.

# Description
- Created three new functions in FishDataProvider.js to filter fish by size.
- Tested via console.log() and verified each of these functions works as intended.
- fish.length types were changed from String to Number and values were modified for this lesson.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
1. `git fetch --all`
2. `git checkout ts-length-divisible-by-3`
3. Open editor of choice and navigate to `FishDataProvider.js`
4. Replace lines 1479, 1490, and 1501 with `console.log(holyFish)`, `console.log(soldierFish)`, and `console.log(nonHolyFish)`, respectively
5. Save changes
6. Navigate to `main.js` and add `holyFish()` to line 7, `soldierFish()` to line 8, and `nonHolyFish()` to line 9
7. Save changes 
8. `serve`
9. Open webpage and developer tools.
10. Check console in developer tools to ensure that `console.log()`is presenting the appropriate data.
